src/transformers/models/trocr/processing_trocr.py

```diff
@@ -58,10 +66,17 @@ def __init__(self, image_processor=None, tokenizer=None, **kwargs):
             raise ValueError("You need to specify a `tokenizer`.")
 
         super().__init__(image_processor, tokenizer)
-        self.current_processor = self.image_processor
         self._in_target_context_manager = False
 
-    def __call__(self, *args, **kwargs):
+    def __call__(
+        self,
+        images: ImageInput = None,
+        *args,
```

Comment:
Not sure we need this `*args`, may lead to silent missing of `text` if it passed not as a keyword argument

Reply:
I see, maybe a better tradeoff would be to remove it or to force users to use keyword arguments for everything other than `images`, I mean use pure `*` in the signature instead of `*args`.

Reply:
We can also add 🚨 for current PR to indicate it is not fully BC